scripts/aggregate-junit-xml.js

```diff
@@ -3,24 +3,28 @@ const fs = require('fs-extra');
 const path = require('path');
 const shell = require('shelljs');
 
+const junitFilesToCheck = [
+  'junit-custom.xml',
+  'junit-custom-unitTests.xml',
+  'junit-custom-integrationTests.xml',
+  'junit-custom-vscodeIntegrationTests.xml'
+];
 const currDir = process.cwd();
 const packagesDir = path.join(currDir, 'packages');
 const dirs = fs.readdirSync(packagesDir).filter(function(file) {
   return fs.statSync(path.join(packagesDir, file)).isDirectory();
 });
-const junitDirs = dirs.filter(dir => {
-  return fs.existsSync(path.join(packagesDir, dir, 'junit-custom.xml'));
-});
+
 shell.mkdir(path.join(process.cwd(), 'junit-aggregate'));
-const actualDirs = junitDirs.map(dir => {
-  const currPackagePath = path.join(
-    process.cwd(),
-    'packages',
-    dir,
-    'junit-custom.xml'
-  );
-  shell.cp(
-    currPackagePath,
-    path.join(process.cwd(), `junit-aggregate/${dir}-junit-custom.xml`)
-  );
+
+dirs.filter(dir => {
+  junitFilesToCheck.forEach(function(junitFile) {
+    var fullFilePath = path.join(packagesDir, dir, junitFile);
+    if (fs.existsSync(fullFilePath)) {
+      shell.cp(
+        fullFilePath,
+        path.join(process.cwd(), `junit-aggregate/${dir}-${junitFile}`)
+      );
+    }
+  });
```

Comment:
For each package directory, check to see if the junit-custom* files are present. If the file is present then copy it to the junit-aggregate folder and prepend the package directory to the file name.